### PR TITLE
[@ember/routing] Add rootURL type to RouterService

### PR DESCRIPTION
**Pull request by gitKrystan**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [docs](https://api.emberjs.com/ember/3.22/classes/RouterService/properties/rootURL?anchor=rootURL), [source](https://github.com/emberjs/ember.js/blob/v3.22.0/packages/%40ember/-internals/routing/lib/services/router.ts#L623)
- [ ] ~If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.~
- [x] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] ~If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.~

### CODE REVIEW

**Review by chancancode — APPROVED** [review]

**Comment by gitKrystan** [comment]
Ready to merge

**Comment by chriskrycho** [comment]
@orta got another bot issue for you: @typescript-bot just merged one without owner approval. 😬

**Review by chriskrycho — COMMENTED** [review]
@gitKrystan good fix!

**Comment by elibarzilay** [comment]
@chriskrycho: this is not changing a critical package, it included tests, so in general the bot accepts any review for it.  It's indicated in the first comment as "... approved by type definition owners, DT maintainers *or others*".

**Comment by chriskrycho** [comment]
Ah. All right, we will take that under advisement; it's understandable but not great.

**Comment by chriskrycho** [comment]
To elaborate: this definitely is a critical package from the point of view of the Ember ecosystem!

**Pull request merged**
